### PR TITLE
groups can be created and they have their own pages

### PR DESCRIPTION
**Pull request by memunyawiri**

### CODE REVIEW

**Review by petrakh — APPROVED** [review]
:shipit:

**Review by paulmillen — APPROVED** [review]
:shipit:

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class GroupsController < ApplicationController
+  def show
+    @group = Group.find(params[:id])
+  end
+  
+  def new
+  end
+
+  def create
+    @group = Group.new(group_params)
```
Comment (dbugsy) [inline app/controllers/groups_controller.rb]: this should be `@group = Group.create(params)`

**Review by dbugsy — APPROVED** [review]
One change, then :shipit: :shipit:

**Pull request merged**